go/internal/feast/onlinestore.go

```diff
@@ -15,8 +15,8 @@ type FeatureData struct {
 }
 
 type OnlineStore interface {
-	// OnlineRead reads multiple features (specified in featureReferences) for multiple
-	// entity keys (specified in entityKeys) and returns an array of array of features,
+	// OnlineRead reads multiple Features (specified in featureReferences) for multiple
```

Comment:
Why are these being capitalized? Our convention thus far has been to use lower case unless we are referring to classes/objects.

Reply:
some autoreplace, will fix